priv/templates/extended_bin

```diff
@@ -163,14 +163,22 @@ PIPE_DIR="${PIPE_DIR:-/tmp/erl_pipes/$NAME/}"
 
 # Extract the target cookie
 COOKIE_ARG="$(grep '^-setcookie' "$VMARGS_PATH" || true)"
+DEFAULT_COOKIE_FILE="$HOME/.erlang.cookie"
 if [ -z "$COOKIE_ARG" ]; then
-    echo "vm.args needs to have a -setcookie parameter."
-    exit 1
+    if [ ! -f "$DEFAULT_COOKIE_FILE" ]; then
+        echo "vm.args needs to have a -setcookie, or $DEFAULT_COOKIE_FILE is required."
+        exit 1
+    elif [ -O "$DEFAULT_COOKIE_FILE" ] && [ $(stat --format=%a $DEFAULT_COOKIE_FILE) -eq 400 ]; then
```

Comment:
The most important is that its permission is just 400.
It isn't 444. And it also isn't 600.

Erlang VM does not allow other than the 400.

However, I don't have an idea.  So, I change to examine only the existence of a file.